hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/metrics/RpcMetrics.java

```diff
@@ -342,6 +344,14 @@ public void incrClientBackoff() {
     rpcClientBackoff.incr();
   }
 
+  /**
+   * Client was backoff due to disconnection
```

Comment:
Is this the other way around, the client ended up being disconnected due to backoffs?

Reply:
done